.github/workflows/test.yml

```diff
@@ -65,6 +65,8 @@ jobs:
     runs-on: ubuntu-latest
     env:
       NUMBER_OF_COMPUTE_NODES: 5
+      NUMBER_OF_THRASH_ITER: 20
+      FILL_PERCENT_CAPACITY: 20
```

Comment:
you should define these locally for the `e2e` step, below the `KUBECONFIG` environment variable - the other steps don't use this.